apps/namadillo/src/hooks/useBalances.ts

```diff
@@ -35,30 +35,32 @@ export const useBalances = (): useBalancesOutput => {
     isSuccess: isStakedBalanceLoaded,
   } = totalStakedBalance;
 
-  const { data: shieldedBalance } = totalShieldedBalance;
-  const isFetchingShieldedBalance = shieldedBalance === undefined;
-  const isShieldedBalanceLoaded = !isFetchingShieldedBalance;
+  const {
+    data: shieldedDollars,
+    isLoading: isLoadingShieldedDollars,
+    isSuccess: isSuccessShieldedDollars,
+  } = shieldedDollarsAmountQuery;
 
   const availableAmount = new BigNumber(balance || 0);
   const bondedAmount = new BigNumber(stakeBalance?.totalBonded || 0);
   const unbondedAmount = new BigNumber(stakeBalance?.totalUnbonded || 0);
   const withdrawableAmount = new BigNumber(
     stakeBalance?.totalWithdrawable || 0
   );
-  const shieldedAmount = new BigNumber(shieldedBalance || 0);
+  const shieldedAmount = shieldedDollars === undefined ? null : shieldedDollars;
   const totalAmount = BigNumber.sum(
     availableAmount,
     bondedAmount,
     unbondedAmount,
     withdrawableAmount,
-    shieldedAmount
+    shieldedDollars || 0
```

Comment:
Hmm, do you think we should make this sum using dollars? Maybe I'm missing something, but I'm not sure it looks correct

Reply:
This is how the Figma layout looks like 🤷 . I'm refactoring this on the next PR that will upgrade the AccountOverview page, but still with the sum of dollars. I believe this is like this because it's doesn't make sense to sum different tokens as they have different values

Reply:
I understand. But in this case you're summing NAM with dollars, right? Maybe we should separate this calc from useBalances and convert the totalAmount to dollars first in order to make the proportion displayed in the chart

Reply:
I understand that until the mainnet we won't be able to query the value in dollars for nam, but we can mock this value temporarily to proceed with the implementation